.travis.yml

```diff
@@ -1,27 +0,0 @@
-language: node_js
-jobs:
-  include:
-    - node: 17
-      env: NODE_OPTIONS=--openssl-legacy-provider
-install:
-  - npm install
-script:
-  - npm run build
-  - npm run test:ci
-  - npm run lint
```

Comment:
Same point as the other PR about adding this line in

Reply:
Added the lint step, and will squash the commits in the merge.